public/index.html

```diff
@@ -8,8 +8,7 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
 
 		<script defer src="https://use.fontawesome.com/releases/v5.0.13/js/all.js" integrity="sha384-xymdQtn1n3lH2wcu0qhcdaOpQwyoarkgLVxC/wZ5q7h9gHtxICrpcaSUfygqZGOe" crossorigin="anonymous"></script>
-	<link rel="stylesheet" href="style.css" media="all">
-
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
```

Comment:
:+1: This thing I was wondering earlier was how the font was loaded. The way this is loaded will fix #84 so all is good!

Reply:
Good point, changed the fontFamily into a constant.